### PR TITLE
Docs say that using code is responsible for show/hide children

### PR DESCRIPTION
**Pull request by almarklein**
Mention in docs that using code is responsible for showing and hiding of child widgets.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "css-loader": "^0.23.1",
     "expect.js": "^0.3.1",
     "istanbul-instrumenter-loader": "^0.1.3",
-    "karma": "0.13.15",
+    "karma": "0.13.21",
```
Comment (blink1073) [inline package.json]: This should be `"^0.13.21"`

Comment (sccolbert) [inline package.json]: can you make this `^0.13.2`? That will remove the merge conflict.

**Comment by almarklein** [comment]
done

**Comment by sccolbert** [comment]
thanks!

**Pull request merged**